packages/d3fc-webgl/src/series/glBar.js

```diff
@@ -0,0 +1,237 @@
+import { rebind } from "@d3fc/d3fc-rebind"
+
+import attributeBuilder from "../buffers/attributeBuilder"
+import uniformBuilder from "../buffers/uniformBuilder"
+import drawModes from "../program/drawModes"
+import programBuilder from "../program/programBuilder"
+import barShader from "../shaders/bar/shader"
+
+export default () => {
+    const program = programBuilder()
+
+    // Attributes.
+    const xValueAttrib = "aXValue", yValueAttrib = "aYValue"
+    const widthValueAttrib = "aWidthValue"
+    // Used to signal if vertex should be moved left = 0.0, or right = 1.0.
+    const directionAttrib = "aDirection"
+    const LEFT = 0.0, RIGHT = 1.0
+
+    // Uniforms.
+    const screenUniform = "uScreen"
+
+    // Builder state.
+    let xScale = null
+    let yScale = null
+    let decorate = program => {}
+    let x = null
```

Comment:
Please can we avoid 1 letter names and where possible align them with what's already used e.g. `xValues`.